content/how-to/exchange/hosted_exchange_mailbox/index.md

```diff
@@ -0,0 +1,170 @@
+---
+permalink: hosted_exchange_mailbox
+audit_date: '2022-12-17'
+title: Hosted Exchange Mailbox Download & FAQ
+type: article
+created_date: '2022-12-17'
+created_by: Rohit Rajput
+last_modified_date: '2022-12-17'
+last_modified_by: Rohit Rajput 
+product: Microsoft Exchange
+product_url: exchange
+---
+
+Hosted Exchange Mailbox Download & FAQ
+========================================
+This article describes how to download your Hosted Exchange mailbox data. Hosted Exchange mailbox data is available in PST format and can be downloaded using the **Rackspace Cloud Office Control Panel** or the **Manage Office 365 at Rackspace Control Panel**. 
+
+Important Considerations for downloading
+--------------------------------------------
+
+Before you proceed to download your Hosted Exchange mailbox data, consider the following points:
+
+ - Ensure that you have Cloud Office Administrator access
+
+ -	Ensure that you have Cloud Office Control Panel & Office 365 Administrator access.
+
+**Note :**
+	You can refer prerequisite terminology, see [Cloud Office support terminology](https://docs.rackspace.com/support/how-to/cloud-office-support-terminology/).
+	
+Downloading Hosted Exchange Mailbox using the Cloud Office Control Panel
+-----------------------------------------------------------------
+**Procedure**
+
+**Step 1.**
+Log in to the Rackspace 
+[Cloud Office Control Panel](https://cp.rackspace.com/identity/Account/Login?ReturnUrl=%2Fidentity%2Fconnect%2Fauthorize%2Fcallback%3Fclient_id%3Dcp3%26redirect_uri%3Dhttps%253A%252F%252Fcp.rackspace.com%252Foidc-login%26response_mode%3Dform_post%26response_type%3Dcode%2520id_token%26scope%3Dopenid%2520offline_access%2520profile%2520email%2520account%26state%3DOpenIdConnect.AuthenticationProperties%253Ddjsyglgcx7CTX136xbcAFZq3XrxRKGal8ikEeE6mBEXEo1repzsQe_I7vu16ypXRrEXWpCZNkW4jxeDQtkvpp0xO53rgIvQtduKp8N-GGjKFptvmgnO1dwVQiCR7zr39uq2WJw%26nonce%3D638069485684794887.Mzg4YzM1MWItOGVkNy00ZjE1LTk5MTgtZDQzZDFkNWFiY2I1ZjU5Y2U2OTctY2VmZi00ZTYzLWEwZmUtZDZkN2ZlZDc4MjRl%26post_logout_redirect_uri%3Dhttps%253A%252F%252Fcp.rackspace.com%252F%26x-client-SKU%3DID_NET451%26x-client-ver%3D5.3.0.0/)
+and Click **Mailboxes** under the **Hosted Exchange** section.
+
+To access Cloud Office Control Panel, using [MyRackspace Portal](https://login.rackspace.com/login), follow the below steps:
+
+**i**.  In the top navigation bar, click **Select a Product>>Dedicated Hosting**.
+
+**ii**.	Click **Products>>Cloud Office**.
+
+**iii**. In the Cloud Office window, click **Open Cloud Office Control Panel**.
+
+**Step 2.** Click **Manage** drop-down menu, next to each mailbox and select **Download Mailbox**. In the Download Mailbox window, enable checkbox to accept the terms and click **Download Mailbox button.**
+
+<img width="468" alt="manage_download_mailbox" src="manage_download_mailbox.png">
```

Comment:
Look at my other PR and you will notice that I just removed the width attribute entirely on each image. This lets the image be whatever size it was saved as. It showed also automatically shrink down smaller when you shrink the browser window and the image should be responsive. If so, then remove the width here and on the others in this file as well.

Reply:
Fix alt tag